packages/docusaurus-theme-classic/src/theme/CodeBlock/WordWrapButton/styles.module.css

```diff
@@ -0,0 +1,25 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+.wordWrapButton {
+  right: calc(
+    var(--ifm-pre-padding) + var(--docusaurus-code-button-size)
+  ) !important;
+}
+
+.wordWrapButtonIcon {
+  width: 1.2rem;
+  height: 1.2rem;
+}
+
+.wordWrapButtonEnabled .wordWrapButtonIcon {
```

Comment:
`wordWrapButtonEnabled` => `wordWrapEnabled`

Reply:
It would break the consistency in naming classes -- for clarity I just take the original class name of a element and add a state name to the end of it. This seems comprehensible, although it looks verbose a bit.

![image](https://user-images.githubusercontent.com/4408379/160692805-986f3e04-e687-4882-8f1c-170ec4f6271e.png)

Reply:
Oh I see, got confused with `codeWithWordWrap` 🤦‍♂️